docs/api/command-line-switches.md

```diff
@@ -48,6 +48,10 @@ Disables the disk cache for HTTP requests.
 
 Disable HTTP/2 and SPDY/3.1 protocols.
 
+### --disable-geolocation-mac
```

Comment:
This can be a generic flag `--disable-geolocation` and document it only for `__macOS__` now which can be extended to other platforms if system services are integrated. I see wip in upstream for something similar on windows.